test/data/css/tests.js

```diff
@@ -209,6 +210,25 @@ exports.test_data = {
             }
         ]
     }, {
+        name: "Preserve Newlines",
+        description: "",
+        matrix: [{
+            options: [
+                { name: "preserve_newlines", value: "true" }
+            ],
+            separator_input: '\\n\\n',
+            separator_output: '\\n\\n',
+        }, {
+            options: [
+                { name: "preserve_newlines", value: "false" }
+            ],
+            separator_input: '\\n\\n',
+            separator_output: '\\n',
+        }],
+        tests: [
+            { input: '.div {}{{separator_input}}.span {}', output: '.div {}{{separator_output}}.span {}' },
```

Comment:
Please add some more tests.  Reusing inputs from other tests is fine, but this needs to show that it will work with a wide array of inputs, including newlines in other locations.